src/iosrv/drpc_client.c

```diff
@@ -43,8 +43,11 @@ notify_ready(void)
 	int		rc;
 
 	req = drpc_call_create(dss_drpc_ctx,
-			DRPC_MODULE_SRV,
-			DRPC_METHOD_SRV_NOTIFY_READY);
+			       DRPC_MODULE_SRV,
+			       DRPC_METHOD_SRV_NOTIFY_READY);
+	if (req == NULL) {
+		return -DER_NOMEM;
+	}
```

Comment:
Not asking for any changes, but AFAIK, for C code, most people (myself included) in the team dislike this "{}" for single-line "if".

Reply:
Understood, though I think there are compelling practical arguments in favor of always using brackets. In particular, there's a long history of security bugs in C code based on developer errors with non-bracketed if-statements. I'd be interested in having a bigger discussion about it in the context of the coding standard.

Reply:
You can discuss this with Liang and other gatekeepers.